varcode/maf.py

```diff
@@ -61,7 +61,7 @@ def load_maf_dataframe(filename, nrows=None, verbose=False):
         filename,
         skiprows=lines_to_skip,
         sep="\t",
-        usecols=range(len(MAF_COLUMN_NAMES)),
+        usecols=list(range(len(MAF_COLUMN_NAMES))),
```

Comment:
I don't know if I'll remember to keep coding like this (constructing every list range twice in Python 2.7.x feels like a wart and something I suspect I'll forget to do).